admin/tools/docker/useful-cmd.sh

```diff
@@ -1,11 +1,11 @@
 # These commands require Docker on your machine and that you belong to 'docker' group
```

Comment:
It's integrated in the sphynx doc, see bottom of `doc/source/install/docker.rst`